Gaijinizer patch/Map402.txt

```diff
@@ -2,512 +2,624 @@
 お……起きた……？
 
 \\N[1]………………くん。
-> CONTEXT: Map402/events/1/pages/0/30/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/30/Dialogue/enemy27
+Y-You're awake, \\N[1]... kun...?#Ron
 > END STRING
 
 > BEGIN STRING
 ご、ごめんね
 
 ちょっとおっぱいで小突くだけのつもりだったんだけど
 い、勢いが強すぎたみたい……
-> CONTEXT: Map402/events/1/pages/0/34/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/34/Dialogue/enemy27
+S-Sorry.
 
+I only meant to poke you a little with my boobs
+but I used a little too much force.#Ron
 > END STRING
 
 > BEGIN STRING
 つ、強すぎるよ！
 衝撃はパンチの域だったよ！！！
 
 そんなにでっかいおっぱいなら………
-> CONTEXT: Map402/events/1/pages/0/39/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/39/Dialogue/kazuya
+T-That was way too much force!
 
+With boobs that big it felt like I was getting
+punched!#Ron
 > END STRING
 
 > BEGIN STRING
 って！！！
 ええええええええええええ！？
 
 な、なんかデカくなってる！？？
-> CONTEXT: Map402/events/1/pages/0/46/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/46/Dialogue/kazuya
+Wait...!!!
+WHAT THE HELL?!
 
+Why are you so big now!??#Ron
 > END STRING
 
 > BEGIN STRING
 い、いや、おっぱいはもともとデカイけど……
 
 じゃなくて！！！なんか全体的に！！！
-> CONTEXT: Map402/events/1/pages/0/51/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/51/Dialogue/kazuya
+I-I mean, your boobs were always pretty big...
 
+No, that's... I mean you're big all over now!!#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 ふ……ふふ………ちがうよ。
 私が大きくなったんじゃないよ…
 
 \\N[1]くんが小さくなったんだよ……
-> CONTEXT: Map402/events/1/pages/0/58/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/58/Dialogue/enemy27
+Fu... Fufu... Not quite.
 
+I haven't gotten any bigger...
+It's you who's gotten smaller...#Ron
 > END STRING
 
 > BEGIN STRING
 エナちゃんの『妖精の針』の魔力を使って
 
 \\N[1]くんの大きさを、
 私たちのサイズまで縮めたの…
-> CONTEXT: Map402/events/1/pages/0/63/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/63/Dialogue/enemy27
+Ena-chan used the magic inside her "Fairy Needle"
+to shrink you down to our size...#Ron
 > END STRING
 
 > BEGIN STRING
 ち、縮めた…！？
-> CONTEXT: Map402/events/1/pages/0/68/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/68/Dialogue/kazuya
+S-Shrink...?!#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]くん……\\C[2]『成長痛』\\C[0]って、知ってる？
 背が伸びたりするとき、脚とかが痛くなるの……
 
 あれ……妖精のイタズラの場合が多いの……
-> CONTEXT: Map402/events/1/pages/0/70/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/70/Dialogue/enemy27
+Do you know what \\C[2]"growing pains" \\C[0]are?
+It's when your legs and other parts of your body
+hurt as your spine grows taller. Fairies often
+play tricks on people during that time.#Ron
 > END STRING
 
 > BEGIN STRING
 エナちゃんみたいな、ごく一部の妖精たちが持ってる
 \\C[2]『妖精の針』………
 
 人間の背を伸ばしたり、縮めたりできる……
-> CONTEXT: Map402/events/1/pages/0/75/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/75/Dialogue/enemy27
+A small fraction of fairies like Ena-chan have
+a \\C[2]"Fairy Needle". It's something they can use to enlarge
+and shrink humans.#Ron
```

Comment:
Line 119 is too long and goes off-screen
"a \\C[2]"Fairy Needle". It's something they can
use to enlarge and shrink humans."